web/views/api/root_view.ex

```diff
@@ -1,65 +1,60 @@
+          href: fix_path(api_show_path(conn, :show, "{id}")),
+          templated: true,
+        }],
+        "prx:sponsors": [%{
+          title: "Get a paged collection of sponsors",
+          profile: "http://meta.prx.org/model/sponsor",
+          href: api_sponsor_path(conn, :index) <> "{?page,per}",
+          templated: true,
+        }],
+        "prx:sponsor": [%{
+          title: "Get info for a single sponsor",
+          profile: "http://meta.prx.org/model/sponsor",
+          href: fix_path(api_sponsor_path(conn, :show, "{id}")),
+          templated: true,
+        }],
 
-      # version: "v1",
-      # _links: %{
-      #   self: %{
-      #     href: api_root_path(conn, :index),
-      #     profile: "http://meta.prx.org/model/api",
-      #   },
-      #   profile: %{
-      #     href: "http://meta.prx.org/model/api",
-      #   },
-      #   "prx:podcasts": [%{
-      #     title: "Get a paged collection of podcasts",
-      #     profile: "http://meta.prx.org/model/metrics/podcast",
-      #     href: api_podcast_path(conn, :index) <> "{?page,per}",
-      #     templated: true,
-      #   }],
-      #   "prx:podcast": [%{
-      #     title: "Get metrics for a single podcast",
-      #     profile: "http://meta.prx.org/model/metrics/podcast",
-      #     href: fix_path(api_podcast_path(conn, :show, "{id}")),
-      #     templated: true,
-      #   }],
-      #   "prx:episodes": [%{
-      #     title: "Get a paged collection of podcast episodes",
-      #     profile: "http://meta.prx.org/model/metrics/episode",
-      #     href: api_episode_path(conn, :index) <> "{?page,per}",
-      #     templated: true,
-      #   }],
-      #   "prx:episode": [%{
-      #     title: "Get metrics for a single podcast episode",
-      #     profile: "http://meta.prx.org/model/metrics/episode",
-      #     href: fix_path(api_episode_path(conn, :show, "{guid}")),
-      #     templated: true,
-      #   }],
-      #   "prx:podcast-downloads": [%{
-      #     href: fix_path(api_podcast_download_path(conn, :index, "{id}") <> "{?interval,from,to}"),
-      #     templated: true,
-      #   }],
-      #   "prx:episode-downloads": [%{
-      #     href: fix_path(api_episode_download_path(conn, :index, "{guid}") <> "{?interval,from,to}"),
-      #     templated: true,
-      #   }],
-      #   "prx:podcast-impressions": [%{
-      #     href: fix_path(api_podcast_impression_path(conn, :index, "{id}") <> "{?interval,from,to}"),
-      #     templated: true,
-      #   }],
-      #   "prx:episode-impressions": [%{
-      #     href: fix_path(api_episode_impression_path(conn, :index, "{guid}") <> "{?interval,from,to}"),
-      #     templated: true,
-      #   }],
-      # }
+      }
+    }
   end
 
-  # defp fix_path(path) do
-  #   path |> String.replace("%7B", "{") |> String.replace("%7D", "}")
-  # end
+  defp fix_path(path) do
+    path |> String.replace("%7B", "{") |> String.replace("%7D", "}")
```

Comment:
This looks familiar!

Reply:
does it look like it was lifted verbatim from castle? :no_mouth: